Decisions.Microsoft365.Exchange/InputExchangeSettings.cs

```diff
@@ -44,6 +44,7 @@ private void OnPropertyChanged(string propertyName)
 
     private string createChildEntityInFolderId;
 
+    [PropertyHidden]
```

Comment:
Need to hide it from mapping as well:

`[PropertyHidden(hiddenInMapping: true)]`

Reply:
Updated @ografov